Packs/ILert/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "iLert",
+    "description": "Alert and notify users using iLert",
```

Comment:
-> "iLert is an IT alerting, on-call management, and uptime monitoring platform that helps DevOps teams respond to incidents faster. Use this iLert content pack to alert and notify users more efficiently."